src/App.js

```diff
@@ -14,11 +16,106 @@ const goodsFromServer = [
   'Garlic',
 ];
 
-const App = () => (
-  <div className="App">
-    <h1>Goods</h1>
-    {goodsFromServer.length}
-  </div>
-);
+export class App extends Component {
+  state = {
+    isLoaded: false,
+    goodsList: [...goodsFromServer],
+    isSorted: true,
```

Comment:
стоит уточнить по какому типу отсортировано
```suggestion
    isSortedByAlphabet: true,
```

the best way
```suggestion
    isSorted: {
      alphabet: false,
      length: false,
    },
```
в целом комменатрий опционален, так как это не часть задания